test/ig/IGExporter.IG.test.ts

```diff
@@ -3119,4 +3119,263 @@ describe('IGExporter', () => {
       ]);
     });
   });
+
+  describe('#r5-properties-on-r4-igs', () => {
```

Comment:
I agree that this is good to test, and I was considering adding a test for each property to ensure that there are not side effects in the output if the property isn't added, but as I thought about it, I think this might be covered by some existing tests. The `should generate an implementation guide for simple-ig` [test](https://github.com/FHIR/sushi/blob/master/test/ig/IGExporter.IG.test.ts#L171) covers the missing `copyrightLabel` and `versionAlgorithm[x]` top level properties, as well as the properties on `resources` and `dependencies`. In the `#customized-ig` describe block, the `should maintain page hierarchy for configured pages` [test](https://github.com/FHIR/sushi/blob/master/test/ig/IGExporter.IG.test.ts#L1117) (as well as a few others in that block) tests the missing properties on `pages`.

While these tests I think do test the functionality you described, it is certainly less direct than it would be if I just added a new test for each case. Before I add in the 5 new tests, I wanted to check if you thought they are beneficial given the other existing tests.

Reply:
Ah, I see what you're saying: since the extensions are only present on R4 IGs, the existing tests for R4 IGs cover this scenario. No new tests are needed just for this.

Reply:
Yep! If we wanted to call out the fact that R4 IGs without R5 properties do not add anything extra, the new tests would do that more explicitly. But with the `toEqual` expectations in those tests, we should catch any unintended consequences if something were to break in the future.